lib/JMAP/Tester/SentenceBroker.pm

```diff
@@ -32,12 +39,47 @@ sub paragraph_for_items {
   });
 }
 
-sub abort_callback { \&abort }
+sub abort {
+  my ($self, $string, $diag_spec) = @_;
+
+  $diag_spec //= [ 'Response sentences', sub { [ $_[0]->items ] } ];
```

Comment:
So we `//=` here, but just below we (if $diag_spec). So I guess you can `abort("...", 0);` ?